app/views/layouts/application.html.erb

```diff
@@ -24,37 +24,47 @@
         <p class="site-title" >Haxxor News</p>
         <nav class="site-nav">
           <div class="trigger">
-              <%= link_to 'Home', root_url, {class: 'page-link'} %>
+            <%= link_to 'Home', root_url, {class: 'page-link'} %>
+            <% if current_user %>
               <%= link_to 'Create Post', new_article_path, {class: 'page-link'} %>
+              <%= link_to 'Sign Out', sessions_path, method: :delete, class: 'page-link-bubble' %>
+            <% else %>
+              <p class="nav-acct">
+                <span class="span-nav">Already have an account? </span>
+                <%= link_to 'Sign In', new_sessions_path, {class: 'page-link-bubble'} %>
+              </p>
+            <% end %>
+
+
           </div>
         </nav>
       </div>
     </header>
-    
+
     <%= yield %>
-    
+
     <footer class="site-footer">
       <div class="wrapper">
         <div class="footer-col-wrapper">
           <div class="footer-col footer-col-1">
-              <p>&copy;2017 by Aatif Jiwani</p>
+            <p>&copy;2017 by Aatif Jiwani</p>
```

Comment:
can replace `2017` with `Date.current.year` or something similar so it's always the current year.